README.md

```diff
@@ -5,186 +5,26 @@
-
-Create a file named `dockercfg.json`. Store your credentials in there; it should look like this:
-
-```hjson
-{
-  // If your cluster does not run on GKE or it runs on GKE and pulls images from other private registries, add the following:
-  "auths": {
-    "gcr.io": {
-      "auth": "BASE64-ENCODED-AUTH-DETAILS"
-    }
-    // Add other registries as necessary
-  },
-  
-  // If your cluster runs on GKE and you are using GCR, add the following:
-  "credHelpers": {
-    "us.gcr.io": "gcloud",
-    "asia.gcr.io": "gcloud",
-    "marketplace.gcr.io": "gcloud",
-    "gcr.io": "gcloud",
-    "eu.gcr.io": "gcloud",
-    "staging-k8s.gcr.io": "gcloud"
-  }
-}
-```
-Finally, create the secret in Kubernetes by running the following command:
-```shell
-kubectl create secret generic snyk-monitor -n snyk-monitor --from-file=./dockercfg.json --from-literal=integrationId=abcd1234-abcd-1234-abcd-1234abcd1234
-```
-
-4. If your private registry requires installing certificates (*.crt, *.cert, *.key only) please put them in a folder and create the following ConfigMap:
-```shell
-kubectl create configmap snyk-monitor-certs -n snyk-monitor --from-file=<path_to_certs_folder>
-```
-
-5. If you are using an insecure registry or your registry is using unqualified images, you can provide a `registries.conf` file. See [the documentation](https://github.com/containers/image/blob/master/docs/containers-registries.conf.5.md) for information on the format and examples.
-
-Create a file named `registries.conf`, see example adding an insecure registry: 
-
-```
-[[registry]]
-location = "internal-registry-for-example.net/bar"
-insecure = true
-```
-
-Once you've created the file, you can use it to create the following ConfigMap:
-```shell
-kubectl create configmap snyk-monitor-registries-conf -n snyk-monitor --from-file=<path_to_registries_conf_file>
-```
-
-## Installation from YAML files ##
-
-The `kubernetes-monitor` can run in one of two modes: constrained to a single namespace, or with access to the whole cluster.
-In other words, the monitor can scan containers in one particular namespace, or it can scan all containers in your cluster.
-The choice of which deployment to use depends on the permissions you have on your cluster.
-
-For _cluster_-scoped deployment you can create the necessary `ServiceAccount`, `ClusterRole`, and `ClusterRoleBinding` required for the monitor's deployment.
-These objects ensure the monitor has the right (limited) level of access to resources in the cluster. The command is as follows:
-```shell
-kubectl apply -f snyk-monitor-cluster-permissions.yaml
-```
-Note that even though the monitor operates in the whole cluster, the `ClusterRole` ensures it can only _read_ or _watch_ resources; the monitor can never modify your objects!
-
-For a _namespaced_ deployment you can create the necessary `ServiceAccount`, `Role`, and `RoleBinding` required for the monitor's deployment:
```

Comment:
I dont think we have this section in the helm docs

Reply:
The first part is a good one to keep, but the second is kind of handled automatically by the Helm chart so no need to create them manually

Reply:
ah ok, we just need to have a section on namespaced deployment in the helm README as I think it is missing